src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,180 @@
+     */
+    private boolean query(final Object value) {
+        return value.equals("FizzBuzz") || value.equals("Fizz") || value.equals("Buzz");
+    }
+
+    /**
+     * A number is fizz if it is divisible by 3 and 5.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage0(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC = array;
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_5 == 0 && (Integer) arrayC[i] % NUMBER_3 == 0 ? "FizzBuzz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+    /**
+     * A number is fizz if it is divisible by 5.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage1(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_5 == 0 ? "Buzz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+
+    /**
+     * A number is fizz if it is divisible by 3.
+     *
+     * @param array int[].
+     * @return Object[]
+     */
+    private Object[] stage2(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_3 == 0 ? "Fizz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+    /**
+     * A number is buzz if it has a 5 in it.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage3(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_10 == NUMBER_5 ? "Buzz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+    /**
+     * A number is buzz if it has a 5 in it.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage4(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : ((Integer) arrayC[i] / NUMBER_10) % NUMBER_10 == NUMBER_5 ? "Buzz"
+                    : arrayC[i];
+        }
+        return arrayC;
+    }
+
+    /**
+     * A number is buzz if it has a 3 in it.
+     *
+     * @param array int[].
+     * @return Object[].
+     */
+    private Object[] stage5(final Object[] array) {
+        Object[] arrayC = array;
+        for (int i = 0; i < arrayC.length; i++) {
+            arrayC[i] = query(arrayC[i]) ? arrayC[i]
+                    : (Integer) arrayC[i] % NUMBER_10 == NUMBER_3 ? "Fizz"
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)